### PR TITLE
Add lenstool recipe, and build scripts

### PR DESCRIPTION
**Pull request by ejullo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ejullo** [comment]
@conda-forge-coordinator, the package passes the test. What is the next step?

**Comment by ejullo** [comment]
@conda-forge-admin , ping @conda-forge/help-python-c for reviewing this this PR

**Comment by ejullo** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by ejullo** [comment]
@conda-forge/help-c ready for review! thanks

File: recipes/lenstool/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - python
+    - libgomp
+    - llvm-openmp
+
+requirements:
+  build:
+    - libtool  # [not win]
+    - {{ compiler('c') }}  # [not win]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
```
Comment (xhochy) [inline recipes/lenstool/meta.yaml:25]: These should be in host

File: recipes/lenstool/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - make  # [not win]
+    - conda-verify
```
Comment (xhochy) [inline recipes/lenstool/meta.yaml:27]: Please don't use this anymore 
```suggestion
```

File: recipes/lenstool/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  host:
+    - python
+    - pip
+    - cfitsio
+    - wcslib
+    - gsl
+    - libopenblas  # [linux]
```
Comment (xhochy) [inline recipes/lenstool/meta.yaml:34]: Would a generic BLAS be ok here?
```suggestion
    - libblas  # [linux]
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/lenstool/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+source:
+  url: https://git-cral.univ-lyon1.fr/{{ name }}/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f14fb546a78ebbae3604739fe70089a98ca22668a85ba0d537fd9d63fc1c0786
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports:
+    - python
```
Comment (xhochy) [inline recipes/lenstool/meta.yaml:16]: This is normally not listed here. Why did you add it?

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/lenstool/meta.yaml (hunk 4 above)
Comment (ejullo) [reply recipes/lenstool/meta.yaml]: I removed this statement from meta.yaml. 

Build passed, but raised 2 warnings related to exports. I guess it's a minor issue though.
<pre>
WARNING (lenstool): run-exports library package conda-forge::llvm-openmp-14.0.6-h61d9ccf_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (lenstool): interpreter (Python) package conda-forge::python-3.10.8-he7542f4_0_cpython in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
</pre>
